### PR TITLE
[SUCE-457] adding new tab groups static pages

### PR DESCRIPTION
**Pull request by SamaneriVimala**

### CODE REVIEW

File: client/elements/sc-page-static.html [hunk 1]
```diff
@@ -167,7 +170,7 @@ <h1 class="title-text">
       </div>
     </app-toolbar>
     <app-toolbar id="nav_toolbar" bottom-item>
-      <template is="dom-if" if="[[!shouldShowSecondToolbar]]">
+      <template is="dom-if" if="[[noToolbarSelected]]">
```
Comment (TrebuhD) [inline client/elements/sc-page-static.html]: I think `noToolbarSelected` would be better suited as a function `isToolbarSelected(shouldShowTipitakaToolbar, shouldShowSecondToolbar, shouldShowOrganizationalToolbar, shouldShowAcademicToolbar)`.

File: client/elements/sc-page-static.html [hunk 2]
```diff
@@ -359,12 +442,26 @@ <h3>{{localize('pageNotFound')}}</h3>
...
               return selector;
           }
 
           _changeView() {
               this.shouldShowSecondToolbar = ['SUBJECTS', 'SIMILES', 'NAMES', 'TERMINOLOGY'].includes(this.selectedPage);
+              this.shouldShowTipitakaToolbar = ['DISCOURSES', 'VINAYA', 'ABHIDHAMMA'].includes(this.selectedPage);
+              this.shouldShowAcademicToolbar = ['NUMBERING', 'ABBREVIATIONS', 'METHODOLOGY'].includes(this.selectedPage);
+              this.shouldShowOrganizationalToolbar = ['ACKNOWLEDGMENTS', 'LICENSING', 'ABOUT'].includes(this.selectedPage);
+              if (this.shouldShowSecondToolbar || this.shouldShowTipitakaToolbar || this.shouldShowAcademicToolbar || this.shouldShowOrganizationalToolbar) {
+                  this.noToolbarSelected = false;
```
Comment (TrebuhD) [inline client/elements/sc-page-static.html]: that way we could get rid of explicitly setting this here.

File: client/localization/elements/sc-page-static/pl.json [hunk 3]
```diff
@@ -11,6 +11,15 @@
     "TERMINOLOGY": "TERMINOLOGIA",
     "SIMILES": "PORÓWNANIA",
     "NAMES": "NAZWY",
-    "SUBJECTS": "TEMATY"
+    "SUBJECTS": "TEMATY",
+    "DISCOURSES": "ROZPRAWIAĆ",
```
Comment (TrebuhD) [inline client/localization/elements/sc-page-static/pl.json]: should be DYSKURSY I think

File: client/localization/elements/sc-page-static/pl.json [hunk 4]
```diff
@@ -11,6 +11,15 @@
...
+    "VINAYA": "PRAWO MONASTYCZNE",
+    "ABHIDHAMMA": "ABHIDHAMMA",
+    "NUMBERING": "NUMERACJA",
+    "ABBREVIATIONS": "SKRÓTY",
+    "METHODOLOGY": "METODOLOGIA",
+    "ACKNOWLEDGMENTS": "PODZIĘKOWANIE",
```
Comment (TrebuhD) [inline client/localization/elements/sc-page-static/pl.json]: PODZIĘKOWANIA - plural

File: client/localization/elements/sc-page-static/pl.json [hunk 5]
```diff
@@ -11,6 +11,15 @@
...
+    "LICENSING": "LICENCJONOWNIE",
```
Comment (TrebuhD) [inline client/localization/elements/sc-page-static/pl.json]: LICENCJONOWANIE

File: client/elements/sc-page-static.html [hunk 6]
```diff
@@ -98,7 +98,10 @@
       }
 
       #nav_tabs,
-      #second_nav_tabs {
+      #second_nav_tabs,
+      #tipitaka_nav_tabs,
+      #academic_nav_tabs,
+      #organizational_nav_tabs {
```
Comment (TrebuhD) [inline client/elements/sc-page-static.html]: this should probably be a single class selector.

File: client/localization/elements/sc-page-static/pl.json (hunk 5 above)
Comment (SamaneriVimala) [reply client/localization/elements/sc-page-static/pl.json]: Oh dear, I guess Google Tranlator failed the test this time!

File: client/elements/sc-page-static.html [hunk 7]
```diff
@@ -342,6 +421,10 @@ <h3>{{localize('pageNotFound')}}</h3>
               }
           }
 
+          _isToolbarSelected(tipitakaToolbar, secondToolbar, organizationalToolbar, academicToolbar) {
+              return (secondToolbar || tipitakaToolbar || academicToolbar || organizationalToolbar) ? false : true;
```
Comment (TrebuhD) [inline client/elements/sc-page-static.html]: `return !(secondToolbar || tipitakaToolbar || academicToolbar || organizationalToolbar);`

**Pull request merged**